lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/ActiveContextManager.cs

```diff
@@ -71,7 +71,7 @@ internal int NumberOfMissingContexts
         /// </summary>
         /// <param name="activeContextObserver"></param>
         /// <returns></returns>
-        public IDisposable Subscribe(IObserver<IDictionary<string, IActiveContext>> activeContextObserver)
+        public IDisposable Subscribe(IObserver<IAllContextsReceived> activeContextObserver)
```

Comment:
ActiveContextManager is an Observable. Theoretically it should implement IObservable<>